src/workspaces/list/RenderedWorkspaces.test.ts

```diff
@@ -39,13 +39,48 @@ describe('The behavior of the RenderedWorkspaces component', () => {
     expect(renderedAzureWS).toHaveLength(1);
   });
 
-  it('should indicate when the workspace is in the process of deleting instead of displaying the description', () => {
+  it.each<{ state: WorkspaceState; statusMessage: string }>([
+    // Cloning and CloningContainer are treated as the same state, for now
+    { state: 'Cloning', statusMessage: 'Workspace cloning in progress' },
```

Comment:
Have I mentioned lately how great parameterized tests are? Thank you to whoever it was that brought them to my attention.

Reply:
I love them too! You can also include the variables in the test case description so that what is printed out when they run differentiates the cases.